### DISCUSSION TITLE
Ability to match windows based on output

### DISCUSSION THREAD

**Discussion by thairanaru** [YaLTeR/niri#648] [Ideas]
I am using a laptop but sometimes I plug a cable to my vertical monitor (well, it's just a regular horizontal monitor which I rotated). However, because the width in the vertical monitor is so small, it feels pointless to have two columns showing in that monitor, and it would be nice to have columns opened maximized by default instead. 
So that's why I am asking window rules to be able to match windows based on what output it is opened.

**Comment by YaLTeR** [comment]
I think you need this: https://github.com/YaLTeR/niri/issues/401
